example/fiber/server.go

```diff
@@ -22,44 +21,35 @@ func main() {
 			endpoint.WithMethod(endpoint.GET),
 			endpoint.WithPath("/product"),
 			endpoint.WithTags("product"),
-			endpoint.WithSuccessfulReturns([]response.Info{response.New([]models.Product{}, "200", "Product List")}),
+			endpoint.WithSuccessfulReturns([]response.Info{
+				response.New("string", "200", "Product List"),                                                        // OK
```

Comment:
I see a lot of these have "OK" on their lines. Did you do manual testing?

Reply:
Yep I was testing in UI and checking them one by one